src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/LegacyCSProjPackageReferenceProjectProvider.cs

```diff
@@ -19,6 +19,9 @@ public class LegacyCSProjPackageReferenceProjectProvider : IProjectSystemProvide
     {
         private readonly IProjectSystemCache _projectSystemCache;
 
+        [Import]
+        private Lazy<ProjectSystem.IProjectServiceAccessor> ProjectServiceAccessor { get; set; }
```

Comment:
sure, I've made it like below. Let me know if that's good.

[Import("Microsoft.VisualStudio.ProjectSystem.IProjectServiceAccessor")]
private Lazy<object> ProjectServiceAccessor { get; set; }